app/src/main/AndroidManifest.xml

```diff
@@ -38,17 +38,35 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
 
-            <!-- firebase dynamic links -->
+            <!-- Airbridge deeplinks -->
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="duckee.abr.ge"
+                    android:scheme="https" />
+            </intent-filter>
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
 
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <data
-                    android:host="duckee.page.link"
+                    android:host="duckee.airbridge.io"
                     android:scheme="https" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="duckee" />
+            </intent-filter>
```

Comment:
Updated the AndroidManifest.xml to include Airbridge deeplink schemes and hosts. This configuration is crucial for the app to intercept and handle Airbridge deeplinks correctly. Without these intent filters, the app won't be able to respond to Airbridge deeplinks.